### PR TITLE
Preview: Add new Android overlay

### PR DESCRIPTION
**Pull request by finagolfin**
I couldn't build packages that depend on this plugin natively on Android otherwise.

### CODE REVIEW

**Comment by finagolfin** [comment]
@d-ronnqvist, tiny pull for review.

File: Plugins/Swift-DocC Preview/SwiftDocCPreview.swift
Comment (d-ronnqvist) [inline Plugins/Swift-DocC Preview/SwiftDocCPreview.swift:1]: Since I don't have a way to check myself; is this the only place that needs this import?

Comment (finagolfin) [reply Plugins/Swift-DocC Preview/SwiftDocCPreview.swift:1]: I don't build this entire repo, but this plugin alone gets pulled in automatically by some Swift packages. Searching this repo, this is the only file that imports `WinSDK`, so most likely the answer is yes.

**Comment by finagolfin** [comment]
@swift-ci please test

**Review by d-ronnqvist — APPROVED** [review]
The code looks good (although I don't have a way to verify that it enables building for Android)

**Pull request merged**